app/lib/actions/katello/repository/sync.rb

```diff
@@ -124,6 +126,62 @@ def regenerate_applicability(repo, contents_changed)
         def rescue_strategy
           Dynflow::Action::Rescue::Skip
         end
+
+        def repository_id
+          input['repository']['id']
+        end
+
+        def repository_name
+          input['repository']['name']
+        end
+
+        def repository_label
+          input['repository']['label']
+        end
+
+        def product_id
+          input['product']['id']
+        end
+
+        def product_name
+          input['product']['name']
+        end
+
+        def product_label
+          input['product']['label']
+        end
+
+        def contents_changed
+          input['contents_changed']
+        end
+
+        def sync_result
+          input['sync_result']
+        end
+        apipie :class, "A class representing #{self} object" do
```

Comment:
Is this tied to `def sync_result` ? Otherwise, a blank line above this would help.

Reply:
Noticed, I've added missing blank lines.